### ISSUE TITLE
Feature - chatgpt o integration with Siri

### ISSUE THREAD

**Issue by moo12cat** [straight-tamago/misakaX#309]

**Comment by dreamwhite** [comment]
I think you don't understand the purpose of this project... It lets enable features that are supported on other models (e.g. AOD) that Apple created

**Comment by moo12cat** [comment]
Yes, because one of the features mentioned at apple wwdc 2024 was integration with gpt and this could be a part of the already existing apple intelligence feature

**Comment by moomoo2402** [comment]
Ahem
